### ISSUE TITLE
Send challenges silently

### ISSUE THREAD

**Issue by albertoxamin** [1inch/shieldy#170]
There are cases in which the bot is added to very large groups that need to have notifications on.
It would be beneficial if in such groups the bot could send messages with the silent feature in telegram.
So when a new user joins it wont bother thousands of the group's members.
This option needs to be passed according to the telegram bot apis `disable_notification`

https://core.telegram.org/bots/api#sendmessage

**Comment by PeterMinin** [comment]
+1
@backmeupplz Could you take a look? What are the maintainers' thoughts on this?

**Comment by backmeupplz** [comment]
it will also not notify the user though

**Comment by PeterMinin** [comment]
True, but not quite. From the docs it seems both the user and everyone else will get a silent notification.
I _think_ this should be enough for the user. Here are my (untested) thoughts. In the most common case, they are already in the chat and will see either the message or a mention button, if they have already scrolled up. If they join and immediately close the chat, they should hopefully see the notification nonetheless. Even a notification with sound can be overlooked, so I don't think a small chance of missing the silent notification is a dealbreaker.

**Comment by backmeupplz** [comment]
it won't be enough, people are dumb
